cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -107,16 +107,20 @@
         }
         for (Map.Entry<String, SlangTestCase> testCaseEntry : testCases.entrySet()) {
             SlangTestCase testCase = testCaseEntry.getValue();
-            if(testCase == null){
+            if (testCase == null) {
                 failedTestCases.put(testCase, "Test case cannot be null");
                 continue;
             }
-            log.info("Running test: " + testCaseEntry.getKey() + " - " + testCase.getDescription());
-            try {
-                CompilationArtifact compiledTestFlow = getCompiledTestFlow(compiledFlows, testCase);
-                runTest(testCase, compiledTestFlow, projectPath);
-            } catch (RuntimeException e){
-                failedTestCases.put(testCase, e.getMessage());
+            if (CollectionUtils.isEmpty(testCase.getTestSuites()) || CollectionUtils.containsAny(testSuites, testCase.getTestSuites())) {
```

Comment:
Since we do not "know" base test suite, I think we want the base test cases (the ones without a test suite specified) to always run.
Do you think o/w?